### PR TITLE
Critical tutorial - Optimizing App Upgrade Processes

### PR DESCRIPTION
**Pull request by jhinkey**
cc/ @ngaskill

@sez11a  - After importing these articles, make sure to move _Modularizing an Existing Portlet_ to be a child of _Modularizing Plugins_. Thanks!

https://issues.liferay.com/browse/LRDOCS-2687

### CODE REVIEW

File: develop/tutorials/articles/02-from-liferay-6-to-liferay-7/08-optimizing-existing-plugins-for-liferay-7/06-modularizing-plugins/00-modularizing-plugins-intro.markdown [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Modularizing Plugins [](id=modularizing-plugins)
+
+This is a placeholder.
```
Comment (sez11a) [inline develop/tutorials/articles/02-from-liferay-6-to-liferay-7/08-optimizing-existing-plugins-for-liferay-7/06-modularizing-plugins/00-modularizing-plugins-intro.markdown]: Can you write something for this? @jorgeferrer has placed a moratorium on placeholders.

Comment (jhinkey) [inline develop/tutorials/articles/02-from-liferay-6-to-liferay-7/08-optimizing-existing-plugins-for-liferay-7/06-modularizing-plugins/00-modularizing-plugins-intro.markdown]: No problem. I already have some text "waiting in the wings". I'll send it your way shortly.

**Comment by jhinkey** [comment]
OK. I filled in that placeholder with content.

**Comment by jhinkey** [comment]
Jorge had reviewed this content a while ago. It's just now we have articles to go under this section parent article.

Since this PR was focused on optimizing upgrade processes, I didn't want to add this to the review. But glad you're up for it. :-)

**Comment by sez11a** [comment]
Hey man, it's critical. :-)

**Comment by sez11a** [comment]
View total diff: https://github.com/sez11a/liferay-docs/compare/35485c88e6...69fcfceb82

**Pull request closed without merging**